sonar-plugin/src/main/java/io/ecocode/javascript/CheckList.java

```diff
@@ -0,0 +1,40 @@
+package io.ecocode.javascript;
+
+import io.ecocode.javascript.checks.*;
+import org.sonar.plugins.javascript.api.JavaScriptCheck;
+import org.sonar.plugins.javascript.api.JavaScriptRule;
+import org.sonar.plugins.javascript.api.TypeScriptRule;
+
+import java.lang.annotation.Annotation;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class CheckList {
```

Comment:
maybe can you merge this class on de repository class ... like done earlier for other languages.
Maybe no added value to split these two classes.

Reply:
In the plugin I have two repositories (for JavaScript and TypeScript) that can use the same checking classes based on how they are annotated. So I considered that it's a good idea to have only one point of truth with the full list